container_engine/django_tutorial/README.md

```diff
@@ -5,187 +5,10 @@ app on Google Container Engine. It uses the [Writing your first Django app](http
 .9/intro/tutorial01/) Polls application as the example app to deploy. From here on out, we refer to this app as
 the 'polls' application.
 
-## Pre-requisites
 
-1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com).
+# Tutorial
+See our [Django on Container Engine](https://cloud.google.com/python/django/container-engine) tutorial for instructions for setting up and deploying this sample application.
```

Comment:
There's secrets in here but the tutorial I see now doesn't have secret anywhere?